src/QMCWaveFunctions/LCAO/AOBasisBuilder.h

```diff
@@ -138,6 +139,16 @@ bool AOBasisBuilder<COT>::put(xmlNodePtr cur)
     addsignforM = 0;
   }
 
+  if (Morder == "Dirac")
+  {
+    expandlm    = DIRAC_CARTESIAN_EXPAND;
+    addsignforM = 0;
+    if (sph != "cartesian")
+    {
+      APP_ABORT(" Error: expandYlm='Dirac' only compatible with angular='cartesian'. Aborting\n");
```

Comment:
Please take the chance to replace all the APP_ABORT in this file.

Reply:
done